mycroft/skills/intent_service.py

```diff
@@ -287,15 +287,18 @@ def handle_utterance(self, message):
                 if not converse:
                     # No conversation, use intent system to handle utterance
                     intent = self._adapt_intent_match(utterances, lang)
+                    padatious_intent = PadatiousService.instance.calc_intent(
+                                        utterances[0])
 
             if converse:
                 # Report that converse handled the intent and return
                 ident = message.context['ident'] if message.context else None
                 report_timing(ident, 'intent_service', stopwatch,
                               {'intent_type': 'converse'})
                 return
-            elif intent:
-                # Send the message to the intent handler
+            elif intent and padatious_intent and padatious_intent.conf < 0.95:
```

Comment:
What if there's no Padatious intent but there is an adapt intent? (Logic should probably be `if intent and not (padatious_intent and padatious_intent.conf < 0.95):`)

Reply:
Ah, yup!  Changing.

Reply:
Except the confidence level comparison needs to be inverted, too:
```python
elif intent and not (padatious_intent and padatious_intent.conf >= 0.95):
```